### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
- [DEMO LINK](https://Zelinskyi-Serhii.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Zelinskyi-Serhii.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Please double check your solution with checklist

File: src/style.css [hunk 1]
```diff
@@ -1 +1,13 @@
...
+  margin-top: 10px;
+}
+
+.fieldset {
+  margin-bottom: 20px;
+}
+
+label:first-of-type {
+  margin-top: 0;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:13]: Screenshot 2023-01-25 at 15 47 28

**Review by OleksiiBlezniuk — APPROVED** [review]
Will approve this time, 'cause in context of this task it's not critical, but don't use tags for styling (only if you need to reset default styles or to set fonts to `html` tag

File: src/style.css (hunk 1 above)
Comment (Zelinskyi-Serhii) [reply src/style.css:13]: Ok.

**Pull request closed without merging**